CHANGELOG.rst

```diff
@@ -38,6 +38,7 @@ Changed
 - narwhalified DropOriginalMixin `#352 <https://github.com/lvgig/tubular/issues/352>_`
 - narwhalified BaseMappingTransformer `#367 <https://github.com/lvgig/tubular/issues/367>_`
 - narwhalified BaseMappingTransformerMixin. As part of this made mapping transformers more 
+- narwhalified DateDifferenceTransformer. `#376 <https://github.com/azukds/tubular/issues/376>_`
```

Comment:
as in the other PR the changelog update seems to be inserted in the middle of another one for some reason?  Not sure what's happened there but maybe try merging in changes form main